daprdocs/content/en/developing-applications/building-blocks/actors/actor-reminders-partitions.md

```diff
@@ -0,0 +1,96 @@
+| `actors\|\|<actor type>\|\|<actor metadata identifier>\|\|reminders\|\|2` | `[ <reminder 1-1>, <reminder 1-2>, ... , <reminder 1-m> ]` |
+| ... | ... |
+
+If the number of partitions is not enough, it can be changed and Dapr's sidecar will automatically redistribute the reminders's set.
+
+## Enabling actor reminders partitioning
+Actor reminders partitioning is currently in preview, so enabling it is a two step process.
+
+### Preview feature configuration
+Before using reminders partitioning, actor type metadata must be enabled in Dapr. For more information on preview configurations, see [the full guide on opting into preview features in Dapr]({{< ref preview-features.md >}}). Below is an example of the configuration:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: myconfig
+spec:
+  features:
+    - name: Actor.TypeMetadata
+      enabled: true
+```
+
+### Actor runtime configuration
+Once actor type metadata is enabled as an opt-in preview feature, the actor runtime must also provide the appropriate configuration to partition actor reminders. This is done by the actor's endpoint for `GET /dapr/config`, similar to other actor configuration elements. Here is a snipet of an actor written in Golang providing the configuration:
+
+```go
+type daprConfig struct {
+	Entities                   []string `json:"entities,omitempty"`
+	ActorIdleTimeout           string   `json:"actorIdleTimeout,omitempty"`
+	ActorScanInterval          string   `json:"actorScanInterval,omitempty"`
+	DrainOngoingCallTimeout    string   `json:"drainOngoingCallTimeout,omitempty"`
+	DrainRebalancedActors      bool     `json:"drainRebalancedActors,omitempty"`
+	RemindersStoragePartitions int      `json:"remindersStoragePartitions,omitempty"`
+}
+
+var daprConfigResponse = daprConfig{
+	[]string{defaultActorType},
+	actorIdleTimeout,
+	actorScanInterval,
+	drainOngoingCallTimeout,
+	drainRebalancedActors,
+	7,
+}
+
+func configHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	json.NewEncoder(w).Encode(daprConfigResponse)
+}
+```
+
+The following, is an example of a valid configuration for reminder partitioning:
+
+```json
+{
+	"entities": [ "MyActorType", "AnotherActorType" ],
+	"remindersStoragePartitions": 7
+}
+```
+
+### Handling configuration changes
+For production scenarios, there are some points to be considered before enabling this feature:
+
+* Enabling actor type metadata can only be reverted if the number of partitions remains zero, otherwise the reminders' set will be reverted to an previous state.
+* Number of partitions can only be increased and not decreased. This allows Dapr to automatically redistribute the data on a rolling restart where one or more partition configurations might be active.
+* This feature is in preview, so the Dapr team is collecting feedback on its correctness and performance prior to enabling it by default.
```

Comment:
Can you add a link to the Community Call the you did. This is the specific URL https://www.youtube.com/watch?v=ZwFOEUYe1WA&t=1457s a the correct time. 
 
Like this
 ## Related Links
* [Actor reminder partitioning community call video](https://www.youtube.com/watch?v=ZwFOEUYe1WA&t=1457s)

Reply:
Done